### PR TITLE
Add navigation bar and header

### PR DESCRIPTION
**Pull request by jenbyte**
- Not styled yet
- Hamburger menu not working yet

### CODE REVIEW

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
I am sure you didn't mean to change 165 files. If the changes you made were small just make a new branch and push your code only. Thank you.

File: client/js/App.js [hunk 1]
```diff
@@ -1,18 +1,12 @@
 import React, { Component } from 'react';
 import { Text, View, StatusBar } from 'react-native';
```
Comment (siddharthparmar7) [inline client/js/App.js]: please remove unnecessary imports.

File: client/js/App.js [hunk 2]
```diff
@@ -1,18 +1,12 @@
...
-// import RootStackNavigator from './navigation/RootStackNavigator';
+import RootStackNavigator from './navigation/RootStackNavigator';
 import Login from './screens/Login';
```
Comment (siddharthparmar7) [inline client/js/App.js]: same as above.

File: client/js/navigation/BottomNavigationLayout.js [hunk 3]
```diff
@@ -1 +1,123 @@
...
+    Card: CardStack,
+    Beers: BeersStack,
+    Events: EventsStack,
+    Stores: StoresStack
+  },
+  {
+    defaultNavigationOptions: ({ navigation }) => ({
+      tabBarIcon: ({ focused, horizontal, tintColor }) => {
+        const { routeName } = navigation.state;
+        console.log('>>>>>>>>>', navigation);
```
Comment (siddharthparmar7) [inline client/js/navigation/BottomNavigationLayout.js]: remove all the console logs.

File: client/js/screens/Card/Card.js [hunk 4]
```diff
@@ -1,14 +1,20 @@
 import React from 'react';
-import { View, Text } from 'react-native';
+import { View, Text, Image, ImageBackground } from 'react-native';
 import PropTypes from 'prop-types';
 import Barcode from 'react-native-barcode-builder';
 import styles from './styles';
 
 const Card = props => {
+  console.log('card');
```
Comment (siddharthparmar7) [inline client/js/screens/Card/Card.js]: remove console logs.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work! Please make the requested changes, and resolve merge conflicts.

Thank you.

File: client/js/navigation/BottomNavigationLayout.js (hunk 3 above)
Comment (jenbyte) [reply client/js/navigation/BottomNavigationLayout.js]: done!

File: client/js/App.js (hunk 2 above)
Comment (jenbyte) [reply client/js/App.js]: done

File: client/js/App.js (hunk 1 above)
Comment (jenbyte) [reply client/js/App.js]: done

File: client/js/screens/Card/Card.js (hunk 4 above)
Comment (jenbyte) [reply client/js/screens/Card/Card.js]: done

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**